tests/modes.spec.ts

```diff
@@ -1,25 +1,132 @@
 import { readFile } from "fs/promises";
 import { test, expect, type Page } from "@playwright/test";
-import { loadInitialPage } from "./shared.ts";
+import {
+  loadInitialPage,
+  clickMap,
+  clearExistingInterventions,
+} from "./shared.ts";
 
 let page: Page;
 
 test.beforeAll(async ({ browser }) => {
   page = await loadInitialPage(browser);
 });
+test.beforeEach(async () => {
+  await clearExistingInterventions(page);
```

Comment:
I forgot this at first and was very confused why tests failed together, but not individually!